Scripts/script-AccessDataWaitForJobResult.yml

```diff
@@ -0,0 +1,76 @@
+commonfields:
+  id: 62e0ac00-6e8b-490c-830c-fbdb9ab2be50
+  version: 1
+name: AccessDataWaitForJobResult
+script: |
+  import json
+  from time import sleep
+
+  timeout = 960
+  interval = 10
+  # Python template - reading arguments, calling a command, handling errors and returning results
+  res = []
+  # Constant and mandatory arguments
+
+  caseid = demisto.get(demisto.args(),'caseid')
+  jobid = demisto.get(demisto.args(),'jobid')
+
+  feDone = False
+
+  ############ Poll stage ############
+  status = None
+  sec = 0
+  ec = {}
+  resp = None
+  sleep(10) # ugly hack to be sure job is submitted, otherwise we get 404
+  while sec < timeout:
```

Comment:
we don't implement polling via scripts. Instead we implement it via playbook. Let's schedule a session and I can show you how to do that.
Or you can also take a look at `Detonate File - JoeSecurity` /  `WildFire - Detonate file`

Reply:
There was a meting  with 'Prasen Shelar' and 'Rony Kozakish' 
I was told that both options are possible, even more I've taken it from one of built-in scripts